codealike.json

```diff
@@ -0,0 +1 @@
+{"projectId":"cc024380-9dc5-11ea-bbbe-0905758693ce","projectName":"tye"}
```

Comment:
What's this file for?

Reply:
That file is related to the code analysis tool I use personally. I had added it to the list in the .gitignore file.

Reply:
Can you remove this file?

Reply:
@davidfowl I have removed the file reference from .gitignore